plugins/koa/index.js

```diff
@@ -0,0 +1,69 @@
+const {transparentGetter, requireFileGetter} = require('@brillout/reconfig/getters');
+
+const packageName = require('./package.json').name;
+
+const serverStartFile = require.resolve('./startServer');
+const koaIntegrationPluginFile = require.resolve('./koaintegrationplugin');
+
+module.exports = {
+    $name: packageName,
+    $getters: [
+        transparentGetter('serverStartFile'),
+        requireFileGetter('koaIntegrationPluginFile'),
+    ],
+    serverStartFile,
+    koaIntegrationPluginFile,
+    ejectables: getEjectables(),
+};
+
+function getEjectables() {
+    const ejectName_server = 'server';
+    const ejectName_serverIntegration = 'server-integration';
+    return [
+        {
+            name: ejectName_server,
+            description: 'Eject the code that creates the Node.js/koa server.',
+            descriptionLong: [
```

Comment:
`descriptionLong` is an experiment. You can remove all `descriptionLong`

Reply:
Done